GrammarLinter.md

```diff
@@ -0,0 +1,67 @@
+## English Grammar Linter (Vale)
+
+This repository includes `beta` rules based on the [Vale grammar linter](docs.errata.ai/vale). To install Vale on a Mac, run:
+
+```
+brew install vale
+```
+
+You can set up a Vale plugin with the VSCode IDE, per https://github.com/errata-ai/vale-vscode. 
+Find these rules in the `styles/` subdirectory. They include grammar rules in .yml format, in several
+subdirectories:
+
+- Modified rules from GitLab in the `styles/cobalt/` subdirectory
+- [Google Developer Style Guide](https://github.com/errata-ai/Google) rules, customized for Vale, in the `styles/Google` subdirectory
+- Rules associated with the [write-good](https://github.com/btford/write-good) grammar linter
+
+These rules are a "Work in Progress"; we may overrule/modify them as we use them to review Cobalt content. For example, if you find a common word / acronym that we use, you're
+welcome to add it (with a PR) to our `styles/cobalt/spelling-exceptions.txt` file.
+
+For more information, see the [Vale documenmtation](https://docs.errata.ai/).
+
+### Vale Configuration
+
+The Vale configuration file is .vale.ini. 
+
+- The `StylesPath` points to rules in the `styles/` subdirectory.
+- The `BasedOnStyles` parameter specifies style subdirectories.
+- The `IgnoredScopes` tells Vale to ignore content as described in [Vale Documentation](https://docs.errata.ai/vale/config#ignoredscopes).
+
+### Use Vale in Your Repository
+
+If you want to use this Vale configuration in your repository, copy:
+
+- `.vale.ini`
```

Comment:
I do not see any `.vale.ini` checked in this PR. Are we supposed to create an empty one ?

Reply:
It already exists on `main`: https://github.com/cobalthq/cobalt-product-public-docs/blob/main/.vale.ini - it looks like this PR is just adding/updating the documentation related to the use of Vale

Reply:
Thank you!